js/game.js

```diff
@@ -80,6 +82,7 @@ class Game {
     } else if (cardOne === 'jack') {
       this.adjustMiddlePile(player)
       updateFeedback('jack', player)
```

Comment:
Instead of calling this main.js function, could you create a key/value pair in the constructor that you change here and have the function in main check for changes? Or something similar?

Reply:
I will create a new branch to work through what we talked about on our call, and see if I can get this to work. Thanks!!